languages/javascript/exercises/concept/array-transformations/.docs/instructions.md

```diff
@@ -0,0 +1,73 @@
+Elyse, magician-to-be, continues her training. She will be given a deck of cards and attempt to transform that deck. She will make certain cards appear, disappear, change their values, or completely rearrange themselves.
+To make things easier, she usually only starts with cards numbered 1 to 10, although some of the tricks may result in larger cards being created.
+
+## 1. - Double every single card
+
+Elyse wants to double the number of every card in the deck. This may result in higher cards than 1-10.
+
+```javascript
+const deck = [1, 2, 3, 4, 10]
+seeingDouble(deck)
```

Comment:
I like the naming! :D

Reply:
Thanks, I tried to make it light and fun.